public/doc/01_icd.md

```diff
@@ -1275,6 +1275,45 @@ When starting the Stream Handler via *systemd* these variables are set by the se
 ![Figure: Sequence to create a transmit stream](./images/MSC_Avb_Control_Establish_Streams.png "Creating a network stream associated with a local source")
 ![Figure: Sequence to remove a transmit stream](./images/MSC_Avb_Control_Remove_Stream.png "Removing a network stream")
 
+# Systemd Watchdog
```

Comment:
Sure that you don't need a
```
#################################################################################
@section watchdog Systemd Watchdog
#################################################################################
```

here? I guess that this section would appear an alien in the final document...

Reply:
Using a 
```
#################################################################################
@section watchdog Systemd Watchdog
#################################################################################
```
would mean that it falls under the _Registries_ Header, wouldn't it ?

Reply:
Nope.